src/lib/Zikula/Bundle/CoreBundle/Twig/Extension/AssetsExtension.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Copyright Zikula Foundation 2015 - Zikula Application Framework
+ *
+ * This work is contributed to the Zikula Foundation under one or more
+ * Contributor Agreements and licensed to You under the following license:
+ *
+ * @license GNU/LGPv3 (or at your option any later version).
+ * @package Zikula
+ *
+ * Please see the NOTICE file distributed with this source code for further
+ * information regarding copyright and licensing.
+ */
+
+namespace Zikula\Bundle\CoreBundle\Twig\Extension;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Zikula\Bundle\CoreBundle\Twig;
+
+use Symfony\Bundle\TwigBundle\Extension\AssetsExtension as Base;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+class AssetsExtension extends Base {
```

Comment:
I'm not a fan of the alias 'Base' because one cannot discern any meaning from the word. In this case something like `SymfonyAssetsExtension` is more descriptive.